serverless/multicloud
# fix: rename test folder to testUtilities

## What did you implement:

Closes [#1091](https://dev.azure.com/711digital/ServerlessApps/_workitems/edit/1091)

We renamed the `core/src/test` to `core/src/testUtilities` because we are having a problem in the content API of 711 with the [serverless-plugin-common-excludes](https://github.com/dougmoscrop/serverless-plugin-common-excludes). This plugin removes all the `**/test` folders from node_modules and this generates an error when we trying to executed the deployed functions.

## How did you implement it:

We renamed the folder and modified all the files that are importing some module from that. Also, we modified the index to export the renamed folder.

## How can we verify it:

We executed the `npm run build` command to verify the correct execution of the tests and tsc:

![image](https://user-images.githubusercontent.com/28662111/65351603-e7c18780-dbbe-11e9-9ba2-7be8294bc5b9.png)


## Todos:

_**Note: Run `npm run test-ci` to run all validation checks on proposed changes**_

       **Validate via `npm test`**
       **Validate via `npm run lint-updated`**
       _Note: Some reported issues can be automatically fixed by running `npm run lint:fix`_
       **Validate via `npm run prettier-check-updated`**
       _Note: All reported issues can be automatically fixed by running `npm run prettify-updated`_

**_Is this ready for review?:_** YES
**_Is it a breaking change?:_** YES
